Sources/SearchableCollectionViewController/SearchBarView.swift

```diff
@@ -1,34 +1,36 @@
 import Foundation
 import UIKit
 
-class SearchBarView: UICollectionReusableView {
-    
-    static let height: CGFloat = 44.0
-    static let kind = String(describing:SearchBarView.self)
-    static let defaultSearchBackgroundColor = UIColor(hex: "00C365")
-    
-    dynamic open var searchBackgroundColor: UIColor? = SearchBarView.defaultSearchBackgroundColor {
-        didSet { searchBar.backgroundColor = searchBackgroundColor }
+open class SearchBarView: UICollectionReusableView {
```

Comment:
Damn, we should update the demo to make sure search is working. Not sure if subclassing is the best way to fix but I guess we'll see when the demo is working. Do you want to fix the demo or should I do it?